### PR TITLE
Fix tests for plugin-server, borked by #4366

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Ref: https://github.com/PostHog/posthog/pull/4366

## Checklist

-   [x] Updated Settings section in README.md, if settings are affected
-   [x] Jest tests

### CODE REVIEW

**Review by Twixes — APPROVED** [review]
Such is life

File: tests/helpers/sql.ts [hunk 1]
```diff
@@ -108,6 +108,7 @@ export async function createUserTeamAndOrganization(
         updated_at: new Date().toISOString(),
         personalization: '{}',
         setup_section_2_completed: true,
+        for_internal_metrics: false,
```
Comment (tiina303) [inline tests/helpers/sql.ts:111]: Why is this needed if the module defaults to false? https://github.com/PostHog/posthog/blob/master/posthog/models/organization.py#L83

Comment (neilkakkar) [reply tests/helpers/sql.ts:111]: .. because Django sets the defaults on the Model object, not the database, so anyone else using raw SQL will bork things :/

Comment (neilkakkar) [reply tests/helpers/sql.ts:111]: wow, it's been on their minds (in some shape or form) for ~15 years: https://code.djangoproject.com/ticket/470
